build/Rulesets/Roslyn.ruleset

```diff
@@ -1,8 +1,115 @@
+    <Rule Id="CA1304" Action="None" /> 
+    <Rule Id="CA1707" Action="None" /> 
+    <Rule Id="CA1710" Action="None" /> 
+    <Rule Id="CA1714" Action="None" /> 
+    <Rule Id="CA1715" Action="None" /> 
+    <Rule Id="CA1716" Action="None" /> 
+    <Rule Id="CA1717" Action="None" /> 
+    <Rule Id="CA1720" Action="None" /> 
+    <Rule Id="CA1721" Action="None" /> 
+    <Rule Id="CA1724" Action="None" /> 
+    <Rule Id="CA1815" Action="None" /> 
+    <Rule Id="CA2007" Action="Warning" /> 
+    <Rule Id="CA2211" Action="None" /> 
+    <Rule Id="CA2218" Action="None" /> 
+    <Rule Id="CA2222" Action="None" /> 
+    <Rule Id="CA2224" Action="None" /> 
+    <Rule Id="CA2225" Action="None" /> 
+    <Rule Id="CA2227" Action="None" /> 
+    <Rule Id="CA2231" Action="None" /> 
+    <Rule Id="CA2234" Action="None" /> 
+  </Rules> 
+  <Rules AnalyzerId="Microsoft.CodeAnalysis.Analyzers" RuleNamespace="Microsoft.CodeAnalysis.Analyzers"> 
+    <Rule Id="RS1001" Action="None" /> 
+    <Rule Id="RS1002" Action="None" /> 
+    <Rule Id="RS1003" Action="None" /> 
+    <Rule Id="RS1004" Action="None" /> 
+    <Rule Id="RS1005" Action="None" /> 
+    <Rule Id="RS1006" Action="None" /> 
+    <Rule Id="RS1008" Action="None" /> 
+    <Rule Id="RS1009" Action="None" /> 
+    <Rule Id="RS1010" Action="None" /> 
+    <Rule Id="RS1011" Action="None" /> 
+    <Rule Id="RS1012" Action="None" /> 
+    <Rule Id="RS1013" Action="None" /> 
+    <Rule Id="RS1014" Action="Warning" /> 
+  </Rules> 
+  <Rules AnalyzerId="Microsoft.Composition.Analyzers" RuleNamespace="Microsoft.Composition.Analyzers"> 
+    <Rule Id="RS0006" Action="Error" /> 
+    <Rule Id="RS0023" Action="Error" /> 
+  </Rules> 
+  <Rules AnalyzerId="Roslyn.Core" RuleNamespace="Roslyn.Core"> 
+    <Rule Id="AD0001" Action="Error" /> 
+  </Rules> 
+  <Rules AnalyzerId="Roslyn.Diagnostic.Analyzers" RuleNamespace="Roslyn.Diagnostics.Analyzers"> 
+    <Rule Id="RS0001" Action="Warning" /> 
+    <Rule Id="RS0002" Action="Warning" /> 
+    <Rule Id="RS0005" Action="Warning" /> 
+    <Rule Id="RS0016" Action="Error" /> 
+    <Rule Id="RS0017" Action="Error" /> 
+    <Rule Id="RS0022" Action="Error" /> 
+    <Rule Id="RS0024" Action="Error" /> 
+    <Rule Id="RS0025" Action="Error" /> 
+    <Rule Id="RS0026" Action="Error" /> 
+    <Rule Id="RS0027" Action="Error" /> 
+  </Rules> 
+  <Rules AnalyzerId="System.Collections.Immutable.Analyzers" RuleNamespace="System.Collections.Immutable.Analyzers"> 
+    <Rule Id="RS0012" Action="Warning" /> 
+  </Rules> 
+  <Rules AnalyzerId="System.Runtime.Analyzers" RuleNamespace="System.Runtime.Analyzers"> 
+    <Rule Id="CA1305" Action="None" /> 
+    <Rule Id="CA1307" Action="None" /> 
+    <Rule Id="CA1308" Action="None" /> 
+    <Rule Id="CA1810" Action="None" /> 
+    <Rule Id="CA1816" Action="None" />
+
+  <!--CA1825 has a bug that fires on compiler generated code and is very noisy in this repo. Reenable this rule when we move to a new version of analyzers.' --> 
```

Comment:
The problem seems to be that the implementation of IOperation on the build of Dev15 that I have doesn't produce the shape expected by the analyzer. I verified that the analyzer doesn't have a bug on Update2. Filed https://github.com/dotnet/roslyn-project-system/issues/241 to track reenabling this.